static/actions_header.html

```diff
@@ -0,0 +1,232 @@
+			
+			-webkit-transform: scale(1);
+			-moz-transform: scale(1);
+			-ms-transform: scale(1);
+			transform: scale(1);
+			
+			-webkit-transition: all 0.3s ease-in;
+			-moz-transition: all 0.3s ease-in;
+			-ms-transition: all 0.3s ease-in;
+			-o-transition: all 0.3s ease-in;
+			transition: all 0.3s ease-in;
+			
+			-webkit-transition-delay: 0s;
+			-moz-transition-delay: 0s;
+			-ms-transition-delay: 0s;
+			-o-transition-delay: 0s;
+			transition-delay: 0s;
+			
+			-webkit-border-radius: 0px;
+			border-radius: 0px;
+		}
+		
+		.knsh-theme .knsh-dashboard-layout-card.knsh-theme-popout:hover {
+			z-index: 1;
+			
+			border-color: #EDEDED;
+			background: #FFFFFF;
+			color: #000000;
+			
+			-webkit-transform: scale(1.03);
+			-moz-transform: scale(1.03);
+			-ms-transform: scale(1.03);
+			transform: scale(1.03);
+			
+			-webkit-box-shadow: 0 0 80px -10px rgba(0,0,0,0.5);
+			box-shadow: 0 0 80px -10px rgba(0,0,0,0.5);
+			
+			-webkit-border-radius: 2px;
+			border-radius: 2px;
+			
+			-webkit-transition-delay: 0.3s;
+			-moz-transition-delay: 0.3s;
+			-ms-transition-delay: 0.3s;
+			-o-transition-delay: 0.3s;
+			transition-delay: 0.3s;
+		}
+		
+		.knsh-theme .knsh-dashboard-layout-group.knsh-theme-accented .knsh-dashboard-layout-card {
+			border-color: #263238;
+			background: #263238;
+			color: #FFFFFF;
+		}
+		
+		.knsh-theme .knsh-dashboard-layout-card.knsh-theme-accented,
+		.knsh-theme .knsh-dashboard-layout-card.knsh-theme-accented:hover {
+			border-color: #263238;
+			background: #263238;
+			color: #FFFFFF;
+		}
+		
+		.knsh-theme .knsh-dashboard-layout-group.knsh-theme-accented .knsh-dashboard-layout-card .knsh-value-visualizer__figure,
+		.knsh-theme .knsh-dashboard-layout-card.knsh-theme-accented .knsh-value-visualizer__figure {
+			text-shadow: 
+				1px 0px 0px #FFFFFF,
+				-1px 0px 0px #FFFFFF,
+				0px 1px 0px #FFFFFF,
+				0px -1px 0px #FFFFFF;
+		}
+		
+		.knsh-theme .knsh-image-visualizer__wrapper {
+			min-height: 200px;
+		}
+	</style>
+</head>
+<body class="knsh-body knsh-sticky-footer knsh-theme">
+	<div class="knsh-sticky-footer__frame">
+		<div class="knsh-sticky-footer__row">
+			<div class="knsh-header-wrapper">
+				<header class="knsh-header">
+					<div class="knsh-columns">
+						<div class="knsh-columns__item">
+							<div class="knsh-header-logo">
+								<a href="/" class="knsh-header-logo__link"><span class="knsh-header-logo__image">TailProduce<sup class="knsh-header-logo__alpha" title="Alpha">&alpha;lpha</sup></span></a>
+								<div class="knsh-header-logo__caption">Backends. Data-centic. Strongly typed. Immutable and append-only. Functional approach. With real-time data crunching. Visualization included.</div>
+							</div>
+						</div>
+						<div class="knsh-columns__item" style="text-align: right;">
+							<a href="https://github.com/KnowSheet/KnowSheet" class="knsh-footer-link"><span>GitHub</span></a>
+						</div>
+					</div>
+				</header>
+			</div>
+		</div>
+
```

Comment:
Add `<div class="knsh-sticky-footer__row knsh-sticky-footer__row__m-expand">` here and the corresponding `</div>` to the `actions_footer.html` (see my comment there).